ajenti/middleware.py

```diff
@@ -2,6 +2,7 @@
 import time
 import random
 import gevent
+import pyotp
```

Comment:
In this case please pack it with stdeb: https://pypi.python.org/pypi/stdeb/0.8.5 and bdist_rpm.

If it has native modules, then separate packages for amd64, i386, armel and armhf are required, otherwise just an universal .deb and .rpm.